File: easy/array/Difference Between Element Sum and Digit Sum of an Array.py
class Solution(object):
    def differenceOfSum(self, nums):
        ele_sum = 0
        dig_sum = 0 
        for i in nums:
            ele_sum += i 
            while i > 0:
                dig_sum += i % 10 
                i/=10 
        return abs(dig_sum - ele_sum)
        

File: easy/array/Difference Between Element Sum and Digit Sum of an Array.cpp
class Solution {
public:
    int differenceOfSum(vector<int>& nums) {
        int ele_sum = 0;
        int dig_sum = 0;

        for(int i=0; i < nums.size(); i++){
            ele_sum += nums[i];
            while ( nums[i] > 0 ){
                dig_sum += nums[i]%10;
                nums[i]/=10;
            }
        }return abs(ele_sum-dig_sum);
    }
};